### ISSUE TITLE
Small confusion in the doc

### ISSUE THREAD

**Issue by scientist1642** [pytorch/pytorch#1134]
http://pytorch.org/tutorials/beginner/blitz/neural_networks_tutorial.html

```
Zero the gradient buffers of all parameters and backprops with random gradients:
net.zero_grad()
out.backward(torch.randn(1, 10))
```
Is there a type of backprop(s) here?

**Comment by apaszke** [comment]
I don't understand the question, can you please rephrase it?

**Comment by scientist1642** [comment]
Sorry, I think I have trouble understanding word **backprop**  in general :) as I usually read in my head:

via backprop - via backpropagation.
let's backprop - let's back propagate gradients.
**backprops with** random gradients - I was confused here.

I'd expect sentence to be:  lets **backprop** (do backpropagation) with random gradient but I don't get where **s** is coming from.

**Comment by apaszke** [comment]
Oh it's just a typo. There's nothing special to it. If you think it's worth fixing we will accept a PR.
